n3fit/src/n3fit/model_trainer.py

```diff
@@ -533,8 +532,8 @@ def _generate_observables(
             force_set_smallest = input_arr.min() > 1e-9
             if force_set_smallest:
                 new_xgrid = np.linspace(
-                        start=1/input_arr_size, stop=1.0, endpoint=False, num=input_arr_size
-                        )
+                    start=1 / input_arr_size, stop=1.0, endpoint=False, num=input_arr_size
```

Comment:
```suggestion
                    start=1/input_arr_size, stop=1.0, endpoint=False, num=input_arr_size
```
just seems a bit odd